src/components/listing-card-prices.js

```diff
@@ -27,76 +31,20 @@ class ListingCardPrices extends Component {
     }
   }
 
-  componentDidMount() {
-    try {
-      this.retrieveConversion()
-    } catch (error) {
-      console.error(error)
-    }
-  }
-
-  doFetch() {
-    let exchangeURL = this.state.exchangeBaseURL
-    exchangeURL += baseCurrencyCode.toLowerCase()
-    exchangeURL += '-'
-    exchangeURL += this.state.currencyCode.toLowerCase()
-
-    return new Promise((resolve) => {
-      fetch(exchangeURL)
-        .then(res => res.json())
-        .then(json => {
-          const exchangeRateFromAPI = json.ticker.price
-          if (typeof Storage !== 'undefined') {
-            const object = { value: exchangeRateFromAPI, timestamp: new Date() }
-            localStorage.setItem('origin.exchangeRate', JSON.stringify(object))
-          }
-          resolve(this.setState({ exchangeRate: exchangeRateFromAPI }))
-        })
-        .catch(console.error)
-    })
-  }
-
-  retrieveConversion() {
-    if (typeof Storage !== 'undefined') {
-      const cachedRate = localStorage.getItem('origin.exchangeRate')
-      if (cachedRate) {
-        const HALF_HOUR = 30 * 60 * 1000
-        const cachedTime = new Date(JSON.parse(cachedRate).timestamp)
-        if (new Date() - cachedTime < HALF_HOUR) {
-          this.setState({ exchangeRate: JSON.parse(cachedRate).value })
-        } else {
-          localStorage.removeItem('origin.exchangeRate')
-          this.doFetch() // cache is invalid
-        }
-      } else {
-        this.doFetch() // isn't cached to begin with
-      }
-    } else {
-      this.doFetch() // localStorage not available
-    }
-  }
-
-  formatApproxPrice() {
-    return Number(this.state.price * this.state.exchangeRate).toLocaleString(
-      undefined,
-      { minimumFractionDigits: 2, maximumFractionDigits: 2 }
-    )
-  }
-
   render() {
     return (
       <div>
         <div className="d-flex align-items-center price-container">
           <div>
             <div className="d-inline-block price placehold">
-              {this.state.exchangeRate == null && (
+              {this.state.fiatPrice == null && (
```

Comment:
Any reason to use `==` vs `===` here?

Reply:
ahh yeah - this was already there and I didn't notice it... pushed a change to `token-ui` to fix it